internal/resolution/resolver_test.go

```diff
@@ -19,112 +18,114 @@ import (
 )
 
 func TestOperatorResolver(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Operator Resolver Suite")
-}
 
-func FakeClient(objects ...client.Object) client.Client {
-	scheme := runtime.NewScheme()
-	if err := v1alpha1.AddToScheme(scheme); err != nil {
-		panic(fmt.Sprintf("error creating fake client: %s", err))
+	testEntityCache := map[deppy.Identifier]input.Entity{"operatorhub/prometheus/0.37.0": *input.NewEntity(
+		"operatorhub/prometheus/0.37.0", map[string]string{
+			"olm.bundle.path": `"quay.io/operatorhubio/prometheus@sha256:3e281e587de3d03011440685fc4fb782672beab044c1ebadc42788ce05a21c35"`,
```

Comment:
This is an actual production image, doesn't looks like there's any reason we can't just use "foo.io/bar/baz"?